client/src/components/App/app.jsx

```diff
@@ -111,7 +111,8 @@ class App extends React.Component {
   }
 
   getAverage (){
-    axios.get(`http://localhost:8080/api/reviews/${this.state.listingId}?type=review`).then(res =>{
+    axios.get(window.location.protocol+'//'+window.location.host+`/api/reviews/${this.state.listingId}?type=review`).then(res =>{
```

Comment:
Why not use back ticks here?

Reply:
i am using it to get ${this.state.listingId}